docker/ubuntu/generate.sh

```diff
@@ -1,11 +1,11 @@
-#!/bin/sh
+#!/busybox/sh
```

Comment:
does ```/usr/bin/env sh``` work?

Reply:
No, their image is too minimal. But I will push a workaround that explicitly calls `sh` instead of changing the shebang.